_getting-started/index.md

```diff
@@ -7,55 +7,117 @@ includeTOC: true
 
 redirect_from: "/getting-started.html"
 ---
+## Try Scala without installing anything 
+You can try Scala, with access to all Scala compilers and 
```

Comment:
```suggestion
You can try Scala in your browser, with access to all Scala compilers and 
```
The "in your browser" is much more engaging, IMO. When looking at other languages, I really like the ability to just try out things in the browser.